web/docs/guides/auth/auth-facebook.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: auth-facebook
+title: "Facebook Login"
+description: Setting up Facebook Login
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Setting up Facebook Login
+
+## Description
+
+Adding a Facebook Login button to your application is easy if you follow these steps.
+
+The setup process consists of three parts:
+
+* Create and configure a Facebook Application on the [Facebook Devalopers Site](https://developers.facebook.com)
+* Add your Facebook keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
+
+## Environment
+
+* [Supabase JS Client](https://github.com/supabase/supabase-js)
+* [Supabase Dashboard (UI)](https://supabase.io)
+* [Facebook Developers Dashboard](https://developers.facebook.com/)
+
+## Requirements
```

Comment:
I think also let's remove this for the initial versions. I see how it could be important later on for some pre-requisites, but a lot if these items are implied (eg, they are using Supabase, we don't need to tell them that they need a Supabase account)